kubernetes/core-svc.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: busybox-service  
+  labels:
+    app: busybox
+spec:
+  ports:
```

Comment:
I cannot see the type of the service. What is the type by default? Could we explicitly define it?

Reply:
ClusterIP is set by default. Yes, it is possible, do I need to change it?

Reply:
I believe ClusterIP suites for our purpose perfectly.